pkgs/by-name/op/optar/package.nix

```diff
@@ -12,7 +12,7 @@ stdenv.mkDerivation {
 
   src = fetchurl {
     url = "http://ronja.twibright.com/optar.tgz";
-    sha256 = "10lr31k3xfcpa6vxkbl3abph7j3gks2210489khnnzmhmfdnm1a4";
+    hash = "sha256-RIVqm6uwfmvhTIiAIISeb8gD71KDrtm3UZe5PmYYmYI=";
   };
```

Comment:
If upstream updates this in‐place and there are no versioned URLs, we should use a Wayback Machine URL for stability. (And presumably bump the `version`, if there are substantive changes?)